.eslintrc.yml

```diff
@@ -55,6 +55,7 @@ overrides:
     rules:
       import/no-extraneous-dependencies: 0
       prettier/prettier: error
+      react/no-unescaped-entities: off
```

Comment:
Not sure what's going on here:
```
  ✘  https://google.com/#q=react%2Fno-unescaped-entities

     `>` can be escaped with `&gt;`


     docs/getting-started/parcel.mdx:67:1
     65 | [You need to use][type=module] `<script src="hello.js" type="module"></script>` to reference `hello.js` from `index.html`, otherwise you will see the following error:
     66 | 
   > 67 | > @parcel/transformer-js: Browser scripts cannot have imports or exports.
        | ^
     68 | 
     69 | </Note>
     70 | 
```
`>` should be compiled to `<blockquote>`?

Reply:
Trying it out in https://mdxjs.com/playground it does compile to `<blockquote>` :thinking: 
/cc @JounQin thought or insights?

Reply:
Markdown inside jsx is still considered as jsx child in `eslint-mdx@v1`, the only workaround is extract the markdown part into another smaller mdx file then import it as Component.

Reply:
Got it, thanks for this explanation!